kymatio/scattering1d/scattering1d.py

```diff
@@ -90,9 +29,12 @@ class Scattering1D(object):
 
         :math:`S_J^{(0)} x(t) = x \\star \\phi_J(t)`,
 
-        :math:`S_J^{(1)} x(t, \\lambda) = |x \\star \\psi_\\lambda^{(1)}| \\star \\phi_J`, and
+        :math:`S_J^{(1)} x(t, \\lambda) =
```

Comment:
We should really really be using $-math in these docstrings.

I had pointed to the `texext` sphinx extension but never added it. Hope to get round to this.

Don't fix it here.

Reply:
Ok. I put this `` :math:`...` `` stuff all over the 1D docstrings (and probably other places too). Will open an issue to address this.